include/seqan/blast/blast_tabular_in.h

```diff
@@ -565,7 +571,13 @@ _readField(BlastMatch<TAlignRow0, TAlignRow1, TPos, TQId, TSId> & match,
             match.sFrameShift = lexicalCast<int8_t>(context._stringBuffer);
             break;
 //         case ENUM::BTOP: write( * ); break;
-//         case ENUM::S_TAX_IDS: write( * ); break;
+        case BlastMatchField<>::Enum::S_TAX_IDS:
+        {
```

Comment:
only cases with local variables have braces so that the local variables are destructed. In this case it doesn't make a difference, but it's a style I have adopted and which is used throughout the file. Changing everything would make the diff a lot larger...